TeensyDmx.cpp

```diff
@@ -728,17 +809,18 @@ uint16_t TeensyDmx::rdmGetSupportedParameters()
         // No sub-devices supported
         return E120_NR_SUB_DEVICE_OUT_OF_RANGE;
     } else {
-        if (m_rdm == nullptr) {
-            m_rdmBuffer.dataLength = 6;
-        } else {
-            m_rdmBuffer.dataLength = 2 * (3 + m_rdm->additionalCommandsLength);
-            for (int n = 0; n < m_rdm->additionalCommandsLength; ++n) {
-                putUInt16(&m_rdmBuffer.data[6+n+n], m_rdm->additionalCommands[n]);
-            }
-        }
+        m_rdmBuffer.dataLength = 8;
         putUInt16(&m_rdmBuffer.data[0], E120_MANUFACTURER_LABEL);
         putUInt16(&m_rdmBuffer.data[2], E120_DEVICE_MODEL_DESCRIPTION);
         putUInt16(&m_rdmBuffer.data[4], E120_DEVICE_LABEL);
+        putUInt16(&m_rdmBuffer.data[6], E120_COMMS_STATUS);
+        if (m_rdm != nullptr) {
+            for (int n = 0; n < m_rdm->additionalCommandsLength; ++n) {
+                putUInt16(&m_rdmBuffer.data[m_rdmBuffer.dataLength+n+n],
```

Comment:
Probably easier to add 2 to dataLength on each of the loop items, then we don't need to do the calculation at the end.

Reply:
Done